dist/components/slider.js

```diff
@@ -791,7 +791,7 @@ $.fn.slider = function(parameters) {
             // (like 35 * .01 = 0.35000000000000003)
             difference = Math.round(difference * precision) / precision;
```

Comment:
Could this line be removed, when the rounding is done later now?

Reply:
I dont know what happen if difference is not rounded on return, I will take a look right now

Reply:
Yes, result seems correct even when JS is doing strange things.